tests/operators/test_plots.py

```diff
@@ -103,7 +104,17 @@ def test_colorbar_map_levels_xaxis(cube, tmp_working_dir):
     cube.rename("zonal_wind_at_pressure_levels")
     cmap, levels, norm = plot._colorbar_map_levels(cube, axis="x")
     assert cmap is None
-    assert (levels == np.linspace(-25, 25, 2)).all()
+    assert levels == [-25, 25]
+    assert norm is None
+
+
+def test_colorbar_map_levels_xaxis_default(cube, tmp_working_dir):
+    """Test for variable without xmin, xmax."""
+    cube = iris.cube.Cube([np.arange(10)])
+    cube.rename("zonal_wind_at_pressure_levels_difference")
```

Comment:
```suggestion
    cube = iris.cube.Cube(np.arange(10), long_name="zonal_wind_at_pressure_levels_difference")
```
The cube can be created with the right name. We also don't need to wrap the data in an outer array.

Reply:
Done.